app/javascript/dashboard/api/specs/contacts.spec.js

```diff
@@ -1,14 +1,58 @@
-import contacts from '../contacts';
+import contactAPI from '../contacts';
 import ApiClient from '../ApiClient';
 
 describe('#ContactsAPI', () => {
   it('creates correct instance', () => {
-    expect(contacts).toBeInstanceOf(ApiClient);
-    expect(contacts).toHaveProperty('get');
-    expect(contacts).toHaveProperty('show');
-    expect(contacts).toHaveProperty('create');
-    expect(contacts).toHaveProperty('update');
-    expect(contacts).toHaveProperty('delete');
-    expect(contacts).toHaveProperty('getConversations');
+    expect(contactAPI).toBeInstanceOf(ApiClient);
+    expect(contactAPI).toHaveProperty('get');
+    expect(contactAPI).toHaveProperty('show');
+    expect(contactAPI).toHaveProperty('create');
+    expect(contactAPI).toHaveProperty('update');
+    expect(contactAPI).toHaveProperty('delete');
+    expect(contactAPI).toHaveProperty('getConversations');
+  });
+
+  describe('API calls', () => {
+    let originalAxios = null;
+    let axiosMock = null;
+    beforeEach(() => {
+      originalAxios = window.axios;
+      axiosMock = {
+        post: jest.fn(() => Promise.resolve()),
+        get: jest.fn(() => Promise.resolve()),
+      };
+      window.axios = axiosMock;
+    });
```

Comment:
Can we make this common helper or something as this is repeated in multiple places?